packages/smooth_app/lib/pages/scan/camera_image_preview.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:smooth_app/helpers/camera_helper.dart';
+import 'package:smooth_app/pages/scan/camera_controller.dart';
+
+/// Forked Widget from the [camera] library, but with a simpler Widget
+class CameraStreamPreview extends StatelessWidget {
```

Comment:
```suggestion
class SmoothCameraStreamPreview extends StatelessWidget {
```

So that it's clear that we have it locally

Reply:
Ok, I will change that @M123-dev.
The PR is right now in draft to let @AshAman999 test if a fix is ok

Reply:
The flash issues seemed to have resolved but the camera was going blank after I take a picture and had to restart my app to start it again, tried it multiple times faced the same issues (for ingredients picture update here in this case )
Android 11, 

https://user-images.githubusercontent.com/57723319/176240606-e6a6dc27-a256-497d-b0d7-e9a1fbf972eb.mp4